exceptions/exceptions.hh

```diff
@@ -144,6 +144,12 @@ struct mutation_write_timeout_exception : public request_timeout_exception {
     { }
 };
 
+// Generating view updates for a single client request can take a long time and might not finish before the timeout is
+// reached. In such case this exception is thrown.
+struct view_update_generation_timeout_exception : public cassandra_exception {
```

Comment:
How is this exception used? We have deliberate mechanisms on the read and write paths to recognize and handle timeout errors and this exception will not be recognized as such. Maybe make `request_timeout_exception` its base class.

Reply:
Alright I moved things around and now `view_update_generation_timeout_exception` derives from `request_timeout_exception`.